samples/appendable-template/ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -0,0 +1,180 @@
+from azureml.pipeline.core.graph import PipelineParameter
```

Comment:
Do we need the original file here? How about replacing it's content with the content of `diabetes_regression_build_train_pipeline_appendable.py`?

Reply:
I keep the original `diabetes_regression_build_train_pipeline.py` because you can compare the original one and the new `diabetes_regression_build_train_pipeline_appendable.py` without accessing to MLOpsPython repo.

Reply:
How about changing filenames?

- diabetes_regression_build_train_pipeline.py --> diabetes_regression_build_train_pipeline_original.py
- diabetes_regression_build_train_pipeline_appendable.py --> diabetes_regression_build_train_pipeline.py

Reply:
I will remove the original version, and rename  `diabetes_regression_build_train_pipeline_appendable.py` to `diabetes_regression_build_train_pipeline.py`.